worker.tf

```diff
@@ -71,10 +74,11 @@ module "additional_worker" {
 
   security_group_ids = concat(
     var.additional_security_group_ids,
-    [exoscale_security_group.all_machines.id]
+    [exoscale_security_group.all_machines.id],
+    var.use_instancepools ? [exoscale_security_group.worker.id] : []
   )
 
-  affinity_group_capacity = var.affinity_group_capacity
+  affinity_group_capacity = 1
```

Comment:
looks strange, is this correct?

Reply:
left over from early testing. Should be a zero diff here, since we handle affinity_group_capacity correctly in the node-group module.